### PR TITLE
[IETF-49] IESG statement page

### PR DESCRIPTION
**Pull request by liamjohnston**
https://springload-nz.atlassian.net/browse/IETF-49

### CODE REVIEW

File: ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html [hunk 1]
```diff
@@ -1,231 +1,79 @@
 {% extends "base.html" %}
```
Comment (liamjohnston) [inline ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html:1]: Basically a copy of the blog page, but with a couple differences... (read on)

File: ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html [hunk 2]
```diff
@@ -1,231 +1,79 @@
...
-            <a class="blog-pagination" href="{{ self.next.url }}">
-                <div class="highlight smaller">
-                    <div class="text">
-                        <p class="sub">
-                           <h3 class="small">Next</h3>{{ self.next.title }}
-                        </p>
-                    </div>
+                <button class="btn btn-outline-primary d-lg-none w-100" type="button" data-toggle="collapse"
+                    data-target="#blog_sidebar" aria-expanded="false" aria-controls="collapseExample">
+                    Statement listing
```
Comment (liamjohnston) [inline ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html:15]: This is one of the differences ("Statement" not "Blog")

File: ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html [hunk 3]
```diff
@@ -1,231 +1,79 @@
...
+            <main role="main" class="col-12 col-lg-8">
+                <div class="px-3 px-xl-5 mb-5">
+                    {% include 'includes/breadcrumbs.html' %}
+                    
+                    {% include "includes/blog_filters.html" with topics=topics parent_url=parent_url %}
+                    {% if filter_text %}
+                        <p class="h6">{{ filter_text }}</p>
+                    {% endif %}
+                    <h1>{{ self.title }}</h1>
+                    <p>{{ self.date|date:"DATE_FORMAT" }}</p>
```
Comment (liamjohnston) [inline ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html:31]: lack of `blog_authors` is another difference

File: ietf/blog/templates/includes/blog_filters.html [hunk 4]
```diff
@@ -110,36 +110,4 @@
...
-        aria-label="Date to"
-        placeholder="Date to"
-        name="date_to"
-        type="text"
-        {% if request.GET.date_to %}value="{{ request.GET.date_to }}"{% endif %}
-    />
-    <div class="input-group-append">
-        <button class="btn btn-outline-primary" type="submit">Filter</button>
-    </div>
-</form> {% endcomment %}
```
Comment (ldanet) [inline ietf/blog/templates/includes/blog_filters.html:145]: Thank you ✨

**Comment by liamjohnston** [comment]
@ldanet one thing to sort out on both the blog template and this one is how the filters/list/breadcrumbs work on mobile. The order and alignment is a bit whack IMO. Let's do that as a separate PR if we have time though.

Screen Shot 2020-10-16 at 2 38 34 PM

File: ietf/blog/templates/includes/blog_filters.html (hunk 4 above)
Comment (liamjohnston) [reply ietf/blog/templates/includes/blog_filters.html:145]: 💅

File: ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html [hunk 5]
```diff
@@ -1,231 +1,79 @@
...
+                    
+                    <p class="intro">{{ self.introduction }}</p>
```
Comment (ldanet) [inline ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html:33]:
```suggestion
                    <p class="lead">{{ self.introduction }}</p>
```
Probably

Comment (liamjohnston) [reply ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html]: how did i manage to not copy and paste that bit 🤔

File: ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html [hunk 6]
```diff
@@ -1,231 +1,79 @@
...
+                    {{ self.prepared_body|safe }}
+                    
+                    {% bibliography self %}
+                    
+                    {% include "includes/social_share.html" only %}
+                    
+                </div>
+                <nav aria-label="blog" class="row no-gutters d-none d-lg-flex">
+                    <div class="col-6 bg-primary p-4 pl-5 d-flex align-items-center">
+                        <a class=" text-white" href="{{ self.previous.url }}">
```
Comment (ldanet) [inline ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html:44]:
```suggestion
                        <a class="text-white" href="{{ self.previous.url }}">
```
My bad 😬

**Review by ldanet — APPROVED** [review]
Looks good to me!

**Pull request merged**